src/components/Articles/ArticlesList.jsx

```diff
@@ -0,0 +1,27 @@
+import ArticlesListArticle from "./ArticlesListArticle.jsx";
+import {useEffect, useState} from "react";
+import {getArticles} from "../../../api.js";
+import {Container} from "react-bulma-components";
+
+export default function ArticlesList() {
+
+    const [articles, setArticles] = useState([])
+    const [isLoading, setIsLoading] = useState(false)
+
+    useEffect(() => {
+        setIsLoading(true)
+        getArticles().then(articles => {
+            setArticles(articles)
+            setIsLoading(false)
+            console.log(articles)
+        })
+    }, []);
+
+    if (isLoading) return <Container alignContent={"center"}>
+        <img className={"image m-auto is-64x64"} src={"src/assets/loading.gif"} alt={"loading symbol"}/>
+    </Container>
+
+    return <div className={"columns is-centered is-multiline"} style={{justifyContent: "stretch"}}>
+        {articles.map(article => <ArticlesListArticle article={article}/> )}
```

Comment:
Unique key is needed for all items in this articles map. Always good to check console in the browser for errors.  It will tell you about these 👍🏼.  There is another error in there about nesting a tags I think!  Be sure to resolve these now as they are hard to debug if you leave them for later.